libpod/container.go

```diff
@@ -1148,3 +1160,70 @@ func (c *Container) copyHostFileToRundir(sourcePath string) (string, error) {
 func (c *Container) StopTimeout() uint {
 	return c.config.StopTimeout
 }
+
+// ParseUser parses the the uid and gid in the format <name|uid>[:<group|gid>]
+// for user flag
+// FIXME: Issue from https://github.com/projectatomic/buildah/issues/66
+func (c *Container) ParseUser(userspec string) (uint32, uint32, error) { //nolint
```

Comment:
Can this be private? I don't think we need it outside of libpod

Reply:
Also, I think this should error if the c.state.Mounted is false, we can't do a lookup if the container is not mounted

Reply:
Removed since we are using chrootuser/GetUser which has same code.